core/arch/arm/plat-nuvoton/utils/npcm845x_trace.c

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2022-2023, ARM Limited and Contributors. All rights reserved.
+ *
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ */
+
```

Comment:
I think you don't need this fle _npcm845x_trace.c_ and _npcm845x_trace.h_. 
Native _lib/libutils/ext/trace.c_ and <_lib/libutils/ext/include/trace.h_ do the job.

Reply:
I have already inserted the code into trace.h and trace.c to support color printing.
If you want me to add explanation into commit, I'll do it